### PR TITLE
[ReadMe] updated readMe file for MediaGallery-MediaStorage modules

### PR DESCRIPTION
**Pull request by vlmed**
### Description (*)
Update README.md file for modules:
- Magento_MediaGallery
- Magento_MediaGalleryApi
- Magento_MediaGalleryCatalog
- Magento_MediaGalleryCatalogIntegration
- Magento_MediaGalleryCatalogUi
- Magento_MediaGalleryCmsUi
- Magento_MediaGalleryIntegration
- Magento_MediaGalleryMetadata
- Magento_MediaGalleryMetadataApi
- Magento_MediaGalleryRenditions
- Magento_MediaGalleryRenditionsApi
- Magento_MediaGallerySynchronization
- Magento_MediaGallerySynchronizationApi
- Magento_MediaGallerySynchronizationMetadata
- Magento_MediaGalleryUi
- Magento_MediaGalleryUiApi
- Magento_MediaStorage

### Fixed Issues (if relevant)

1. magento/devdocs magento/devdocs#4451: README content for modules - work in M2 Code

### Contribution checklist (*)
 - [ ] Pull request has a meaningful description of its purpose
 - [ ] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds are green)

### CODE REVIEW

File: app/code/Magento/MediaGalleryApi/README.md [hunk 1]
```diff
@@ -2,12 +2,56 @@
...
+For information about module installation in Magento 2, see [Enable or disable modules](https://devdocs.magento.com/guides/v2.4/install-gde/install/cli/install-cli-subcommands-enable.html).
+
 ## Extensibility
 
-Extension developers can interact with the Magento_MediaGallery module. For more information about the Magento extension mechanism, see [Magento plug-ins](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/plugins.html).
+Extension developers can interact with the Magento_MediaGalleryApi module. For more information about the Magento extension mechanism, see [Magento plug-ins](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/plugins.html).
+
+[The Magento dependency injection mechanism](https://devdocs.magento.com/guides/v2.4/extension-dev-guide/depend-inj.html) enables you to override the functionality of the Magento_MediaGalleryApi module.
+
+### Public APIs
```
Comment (bgorski) [inline app/code/Magento/MediaGalleryApi/README.md:15]: Please also add SearchAssetsInterface

**Review by bgorski — CHANGES_REQUESTED** [review]

**Comment by bgorski** [comment]
@vlmed thank you for your contribution! I had just one comment, the rest looks good to me. Please also resolve conflicts and we can proceed with this one.

**Review by bgorski — APPROVED** [review]

**Comment by bgorski** [comment]
@vlmed approved, thank you for the contribution!

**Comment by engcom-Charlie** [comment]
:heavy_check_mark: QA Passed

The PR request has updation related to the **_README.md_** file content only.

**No other manual testing is required** and there is **no impact on regression testing**.

**Comment by BarnyShergold** [comment]
@magento run all tests

**Comment by BarnyShergold** [comment]
@magento run WebAPI Tests

**Comment by BarnyShergold** [comment]
@magento run Functional Tests B2B

**Comment by BarnyShergold** [comment]
@magento run Functional Tests B2B

**Comment by BarnyShergold** [comment]
@vlmed - All the tests have passed but the CLA seems stuck or not correct. I'm assuming that you have signed this? One way to kick this (we use this on DEVDOCS) is to Close then reopen the PR - what do you want to try?

**Review by sidolov — APPROVED** [review]

**Comment by magento-engcom-team** [comment]
Hi @sidolov, thank you for the review.
[ENGCOM-9229](https://jira.corp.magento.com/browse/ENGCOM-9229) has been created to process this Pull Request
:eight_spoked_asterisk: @sidolov, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Pull request merged**